mitodl/ol-data-platform
# remove salesforce dbt models

### Description (What does it do?)
Salesforce passwords have expired https://airbyte.odl.mit.edu/workspaces/3f0ccc59-baa3-4567-92e5-c61fb737467b/connections/ad3f786c-54b7-4aa9-a29e-521c3a47af72/job-history#8746::0
We need to pause that pipeline. This ticket is to remove all the staging and intermediate salesforce dbt models we currently have. This is an urgent need.

### How can this be tested?
Check to see all salesforce dbt models are now missing (see ol_warehouse_production_kdelaney_staging and ol_warehouse_production_kdelaney_intermediate). And run:

dbt build --select int__mitxpro__b2becommerce_b2border
